docs/core-concepts/Planning.md

```diff
@@ -23,6 +23,25 @@ my_crew = Crew(
 
 From this point on, your crew will have planning enabled, and the tasks will be planned before each iteration.
 
+#### Planning LLM
+
+Now you can define the LLM that will be used to plan the tasks. The default LLM is `gpt-4o-mini`, but you can change it to any other LLM as long as it is compatible.
```

Comment:
We should drop this part of the description and change it to say they can use any ChatOpenAI llm:

`The default LLM is `gpt-4o-mini`, but you can change it to any other LLM as long as it is compatible.`

Reply:
Changed 👍🏽